flask_mailgun.py

```diff
@@ -101,27 +179,33 @@ def send(self, **data):
         from_ = data.pop('from_', None)
         if from_ is None:
             from_ = 'no-reply@{}'.format(self.domain)
+
         data['from'] = from_
+
         if 'html' in data and 'text' not in data:
             data['text'] = html2text(data['html'])
+
         if self.debug:
             data.setdefault('html', '(no HTML)')
             current_app.logger.debug(debug_template, data)
             return
-        url = 'https://api.mailgun.net/v3/{}/messages'.format(self.domain)
```

Comment:
This is the official Mailgun API endpoint URL. Why would you need a different one? Different version perhaps? In that case, I think it would be more convenient to have just the version in configuration.